src/stubServer.ts

```diff
@@ -102,54 +101,60 @@ async function parseConfig(apiPath: string, req: express.Request) {
   return name;
 }
 
+type ExpressHandlerFunction = (req: express.Request, res: express.Response) => void;
+
 async function processStubRequest(
   apiPath: string,
   req: express.Request,
   res: express.Response,
   next: express.NextFunction
 ) {
-  const stubName = await parseConfig(apiPath, req);
+  try {
+    const stubName = await parseConfig(apiPath, req);
 
-  if (isUrl(stubName)) {
-    const url = stubName;
-    send(url, req, res, next);
-  } else {
-    const filename = stubName;
-
-    // eslint-disable-next-line @typescript-eslint/ban-types
-    let fileContent: string | object | undefined;
-
-    if (filename.endsWith('.json') || filename.endsWith('.js') || filename.endsWith('.ts')) {
-      // Can load .json, .js or .ts files
-      // If file does not exist: "Cannot find module '...'"
-      deleteRequireCache(filename);
-      const jsFileContent = (await import(filename)).default;
-
-      if (
-        (filename.endsWith('.js') || filename.endsWith('.ts')) &&
-        typeof jsFileContent === 'function'
-      ) {
-        // Assume the function is an Express request handler
-        jsFileContent(req, res);
-      } else {
-        fileContent = jsFileContent;
-      }
+    if (isUrl(stubName)) {
+      const url = stubName;
+      send(url, req, res, next);
     } else {
-      // Anything else: .html, .jpg...
-      // If file does not exist: "ENOENT: no such file or directory, open '...'"
-      fileContent = fs.readFileSync(filename);
-    }
+      const filename = stubName;
 
-    if (fileContent !== undefined) {
       const match = /_(\d+)_[A-Za-z]+/.exec(filename);
       const httpStatus = match !== null ? Number(match![1]) : 200; // 200: default HTTP status if none specified
 
       if (httpStatus === 204 /* No Content */) {
         res.status(httpStatus).end();
       } else {
-        res.status(httpStatus).send(fileContent);
+        let fileContent: ExpressHandlerFunction | object | Buffer | undefined;
+
+        if (filename.endsWith('.json') || filename.endsWith('.js') || filename.endsWith('.ts')) {
```

Comment:
why didn't you use https://nodejs.org/api/path.html#pathextnamepath ?

Reply:
Because I didn't know about it.

```TypeScript
if (['.json', '.js', '.ts'].includes(path.extname(filename))) {
  ...
}
```